src/lib/finishPollMessage.ts

```diff
@@ -78,11 +78,13 @@ export async function finishPollMessage({
             .getEnvironmentReader()
             .getSettings()
             .getById('wordcloud-api');
-        const timeZone = await read
-            .getEnvironmentReader()
-            .getSettings()
-            .getById('timezone');
 
+        const user = await read.getUserReader().getById(data.user.id);
+        const utcOffset = await user.utcOffset;
+        const usertime = await timeZones.timeZones.find(time => time.offset === utcOffset) as any;
```

Comment:
Please avoid using `any` as much as possible. That's why we're using "type"script and not javascript :)
 Here I think you can check for undefined values, or perhaps declare a type for the expected return value.

Reply:
Ok, sir ill keep this in mind and give a proper type definition!